scripts/tests/AUTH/test_3d_phase3_config_integrity.py

```
@@ -0,0 +1,327 @@
+        token = None
+
+    if not token:
+        print("[ERROR] Admin auth response missing session_token")
+        return None
+
+    return token
+
+
+def run_sqlite(sql: str, db_path: str) -> str:
+    cmd = [*COMPOSE_ARGS, "exec", "-T", "backend", "sqlite3", "-readonly", db_path]
+    result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)